.evergreen/config.yml

```
@@ -1120,6 +1120,39 @@ buildvariants:
   - name: publish-packages
     distros:
     - rhel70-small
+- name: amazon2023
+  display_name: "Amazon Linux 2023"
+  run_on: amazon2023.0-large
```

```suggestion
  run_on: amazon2023.0-small
```

I suggest using the `-small` distros for libmongocrypt tasks. I expect these tasks are not resource intensive (compared with server build or test tasks) and may not benefit much from the `-large` distros.

Comment applies to all new variants.